hydra-node/test/Hydra/Network/AuthenticateSpec.hs

```diff
@@ -120,6 +121,8 @@ spec = parallel $ do
     prop "can roundtrip CBOR encoding/decoding of Signed Hydra Message" $
       prop_canRoundtripCBOREncoding @(Signed Msg)
 
+    roundtripAndGoldenSpecs (Proxy @AuthLog)
```

Comment:
You might want to commit the golden file.

Reply:
🤦